src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -377,6 +378,12 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
         )
         pop = self.initialize_medication_coverage(pop)
 
+        # Generate lifestyle adherence column
+        lifestyle_propensity = self.randomness.get_draw(pop_data.index)
```

Comment:
Don't we need an additional key here?

Reply:
I was able to get uniformly distributed draws and create an adherence column with the expected ratio without it but I'll add one.